Client/Application/AppDelegate.swift

```diff
@@ -484,6 +484,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 self.launchFromURL(params)
             }
         }
+
+        guard let extensionDefaults = UserDefaults(suiteName: "group.org.mozilla.ios.Fennec") else { return }
+        var numberOfBookmarks = extensionDefaults.integer(forKey: "numberOfBookmarksFromShareToExtension")
+        while numberOfBookmarks > 0 {
```

Comment:
This can also be a simple 3 line `for` loop that is a bit more concise.  I don't know if this is a good idea becaue `track()` will do a `DispatchQueue.main.async(execute:)` - so what happens if you have multiple bookmarks. will they all run at the same time?